### ISSUE TITLE
Update Testimonials to Carousel

### ISSUE THREAD

**Issue by ShaanCoding** [ShaanCoding/makeread.me#102]
## Feature Details - Low priority task

Uplift the testimony section to a carousel. Have left & right buttons with lucide react. Have drag functionality and loop when reaching the end. Use framer-motion for the animation.

- Also have the middle card pop out more then the left & right cards
- Be responsive
- Potentially move the testimony data / content to backend after #87

**Comment by ShaanCoding** [comment]
Not needed tbh
